lib/main.dart

```diff
@@ -31,6 +33,8 @@ class _RandomWordsState extends State<RandomWords> {
   final _suggestions = <WordPair>[];
   final _saved = <WordPair>{};
   final _biggerFont = const TextStyle(fontSize: 18.0);
```

Comment:
I'd remove ```_biggerFont``` cause you're only using it once, and the style is already pretty straightforward

Reply:
I wouldn't be so convenient about this, because if i would scale this state in the future this font might be reused

Reply:
you probably meant *confident* instead of *convenient* (I don't recommend being *convenient* in general, as people will try to abuse this)